zcash_client_sqlite/src/wallet.rs

```diff
@@ -3030,6 +3030,11 @@ pub(crate) fn set_transaction_status(
             conn.execute(
                 "UPDATE transactions
                  SET mined_height = :height,
+                     min_observed_height = MIN(
+                        min_observed_height,
+                        IFNULL(mined_height, :height),
+                        :height
+                     ),
```

Comment:
`MIN(NULL, ...)` returns NULL, which I think is not what you want here if `min_observed_height` was previously NULL.

Reply:
Per `zcash_client_sqlite/src/wallet/db.rs`:
https://github.com/zcash/librustzcash/blob/32f8ad963c1d2a62e03236c2daa0bcefa2daa1c5/zcash_client_sqlite/src/wallet/db.rs#L286